config.yml

```
@@ -4506,6 +4514,7 @@ nodes:
       - name: arguments
         type: node?
         kind: ArgumentsNode
+        comment: "Can be `nil` when there are empty parentheses, like `super()`."
```

Only? Any other case and it would be `ForwardingSuperNode` instead, right?

```suggestion
        comment: "Can be `nil` only when there are empty parentheses, like `super()`."
```

---

That's correct